website/docs/events/multiplier-event-1.md

```diff
@@ -0,0 +1,43 @@
+# Multiplier Event 1: Pre-launch of 1st content
+
+- Date: 27.10.2022
+- Time: 9:00 - 14:00 EEST
+- Place: Room PR 105, Faculty of Automatic Control and Computer Science, POLITEHNICA University of Bucharest
+
+## Initial Presentation: 9:00 - 11:00
+
+The team and community members will hold presentations regarding the purpose and aims of Open Education Hub, our mission and ideals.
+The team will go over the following topics:
+
+- Overview of the project (Răzvan Deaconescu)
+- Methodology (Răzvan Nițu)
+- Content (Teodor Duțu)
+- Infrastructure (Sergiu Weisz)
+- Community (Liza Babu)
+
+Each talk aims to take no more than 20 minutes, followed by 5 - 10 minutes for questions and discussions.
+
+## Coffee Break: 11:00 - 11:30
+
+This will be an opportunity for more in-depth and informal discussions regarding the topics presented so far.
+
+## Demo from the Open Education Hub Team: 11:30 - 12:00
+
+In this part we will showcase how to create a repository under the [Open Education Hub organization](https://github.com/open-education-hub/) using our templates and how to contribute to it via pull requests and in accordance to our methodology.
+
+## Practical Workshop: 12:00 - 13:00
+
+Participants will create their own repositories within the the [Open Education Hub organization](https://github.com/open-education-hub/) and start developing content in order to familiarize themselves with our workflow.
```

Comment:
I wouldn't create them in the organization. They can be "hosted" by Open Education Hub once they pass certain criteria (that we have yet to write down).

Reply:
I would reword this as participants are "encouraged". And tell they would require a laptop. (Wireless) Internet access will be provided.

But participants can also just help others, or engage the Open Education Hub team. I wouldn't "require" them to create content / repositories.

I think the most important part is for them to reproduce the build / deployment of the Operating Systems class. That I think is the most important part. The creation of another repository I would mark as optional.